hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -6,7 +6,7 @@ This tutorial explains how to boot your Raspberry Pi from a USB mass storage dev
 
 ## Program USB boot mode
 
-The Raspberry Pi 3+ is able to boot from USB without any changes, but the Raspberry Pi 3 requires the USB boot bit to be set in the OTP (one-time programmable) memory. If you are using a Raspberry Pi 3+, please go to the next section.
+The Raspberry Pi 3*+ is able to boot from USB without any changes, but the Raspberry Pi 3* requires the USB boot bit to be set in the OTP (one-time programmable) memory. If you are using a Raspberry Pi 3*+, please go to the next section.
```

Comment:
Change 3*+ to '3A+ and 3B+'

Change 3* to '3A and 3B'

as James says.

Reply:
Ok